script.js

```diff
@@ -0,0 +1,203 @@
+const checkingForValidation = (date) => {
+  if (date === undefined || date === null) {
+    return true;
+  }
+  return false;
+};
+
+/*Cтворити функцію, яка створює та реєструє нову команду у автопробізі.
+Під час виклику функції потрібно передати дані про команду - назва команди, ім'я водія, рік народження водія, марка автомобіля,
+спонсор команди та чи водій допущений до автопробігу*/
+
+const avtoprobig = () => {
+  nameComand = prompt(`Please enter a name of your comand`);
+  nameOfDriver = prompt(`Please enter a name of driver`);
+  yearOfBirthday = prompt(`Please enter a year of driver's birthday`);
+  carBrand = prompt(`Please enter a car brand`);
+  sponsorOfComsnd = prompt(`Please enter is sponsor of your comand`);
+  isAllowedToDrive = prompt(`Please enter this driver allowed to avtoprobig`);
+
+  if (
+    checkingForValidation(nameComand) ||
+    nameComand === "" ||
+    checkingForValidation(nameOfDriver) ||
+    nameOfDriver === "" ||
+    checkingForValidation(yearOfBirthday) ||
+    yearOfBirthday === "" ||
+    checkingForValidation(carBrand) ||
+    carBrand === "" ||
+    checkingForValidation(sponsorOfComsnd) ||
+    sponsorOfComsnd === "" ||
+    checkingForValidation(isAllowedToDrive)
+  ) {
+    return console.log(
+      "You entered incorrect information. Please, enter information one more."
+    );
+  } else {
+    let newComand = {
+      nameComand,
+      nameOfDriver,
+      yearOfBirthday,
+      carBrand,
+      sponsorOfComsnd,
+      isAllowedToDrive,
+    };
+    console.log(newComand);
+  }
+};
+avtoprobig();
+
+/* Запитати дані (через prompt та циклічно) про нового користувача у системі та зберегти його в об'єкт.
+Дані про користувача - логін, пароль, місто, країна, стать, вік*/
+
+let login = prompt(`Please, enter your login.`);
+let password = prompt(`Please, enter your password.`);
+let city = prompt(`Please, enter your city.`);
+let country = prompt(`Please, enter your country.`);
+let sex = prompt(`Please, enter your sex.`);
+let age = prompt(`Please, enter your age.`);
+let user;
+if (
+  checkingForValidation(login) ||
+  login === "" ||
+  checkingForValidation(password) ||
+  password === "" ||
+  checkingForValidation(city) ||
+  city.length < "" ||
+  checkingForValidation(country) ||
+  country === "" ||
+  checkingForValidation(sex) ||
+  sex === "" ||
+  checkingForValidation(age) ||
+  age === ""
+) {
+  console.log(
+    `You entered incorrect information. Please, enter information one more.`
+  );
+} else {
+  user = {
+    login,
+    password,
+    city,
+    country,
+    sex,
+    age,
+  };
+}
+
+// console.log(user);
+
+/*Створити функцію, яка буде міняти дані в конкретного користувача створеного пунктом вище.
+Наприклад сhangeUserData(user_1, city, 'Kherson). Де user_1 - обʼєкт в якому буде мінятись, city - поле,
+яке буде мінятися на нове значення - 'Kherson'*/
+
+let user_1 = user;
+
+const сhangeUserData = () => {
```

Comment:
According to the task, you should pass two params in the function, the object that will change and the exact field to change, and the value
It should be something like

```suggestion
const сhangeUserData = (obj, field, value) => {
    return obj.field = value
}
```

Also nice to have validation, to verify if the field exists and passed data at least not empty